editor/src/core/shared/template-path.ts

```diff
@@ -914,3 +914,35 @@ export function staticScenePathContainsElementPath(
     return elementPathsEqual(sceneElementPath, elementPath)
   })
 }
+
+export function isScenePathEmpty(path: TemplatePath): boolean {
+  if (isScenePath(path)) {
+    return path.sceneElementPaths.length === 0
+  } else {
+    return path.scene.sceneElementPaths.length === 0
+  }
+}
+
+interface DropFirstScenePathElementResultType {
+  newPath: InstancePath
+  droppedScenePathElements: StaticElementPath | null
+}
+
+export function dropFirstScenePathElement(
+  path: StaticInstancePath,
+): DropFirstScenePathElementResultType {
+  if (path.scene.sceneElementPaths.length === 0) {
```

Comment:
Should this use `isScenePathEmpty`?

Reply:
It probably should actually, it did, then this function changed a couple of times and now it probably can again.

Reply:
Fixed in 9bf80891.